src/Compilers/Core/Portable/CommandLine/CommonCompiler.cs

```diff
@@ -568,7 +566,7 @@ internal bool ReportDiagnostics(IEnumerable<DiagnosticInfo> diagnostics, TextWri
         /// non-error diagnostic were later elevated to an error through filtering (e.g., through
```

Comment:
This statement seems wrong. The above explicitly says that these are only unsuppressable errors, so anything elevated to an error wouldn't affect this check.